docs/guide/extending_online_editor.md

```diff
@@ -255,6 +255,85 @@ In the example above, the template files for the front end could be:
 
 Templates for Content View in the Back Office would be `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_word.html.twig` and `app/Resources/views/themes/admin/field_type/ezrichtext/custom_style/highlighted_block.html.twig` respectively (assuming Admin SiteAccess uses the `admin` theme).
 
+## Custom attributes and classes
+
+You can add custom attributes and CSS classes to elements in the Online Editor.
+
+The available elements are:
+
+- `embedinline`
+- `embed`
+- `formatted`
+- `heading`
+- `embedimage`
+- `ul`
+- `ol`
+- `li`
+- `paragraph`
+- `table`
+- `tr`
+- `td`
+
+### Custom attributes
+
+Custom attributes are configured under the `fieldtypes.ezrichtext.attributes` key.
+
+A custom attribute can belong to one of four types: `choice`, `boolean`, `string`, or `number`.
```

Comment:
Nitpick (if I'm right):
```suggestion
A custom attribute can belong to one of four types: `choice`, `boolean`, `string` or `number`.
```
Do we place comma before "or"?

Reply:
We can (and want to 😉 ), ref. https://en.wikipedia.org/wiki/Serial_comma

Reply:
:scream: good to know :ok_hand: